scripts/jobs/housing/housing_mtfh_case_notes_enriched.py

```diff
@@ -0,0 +1,109 @@
+import boto3
+from datetime import datetime, timedelta, timezone, date
+import logging
+import pandas as pd
+import spacy
+from spacy.cli import download
+download("en_core_web_sm")
+from spacy.lang import punctuation
+from spacy.lang.en.stop_words import STOP_WORDS
+
+from scripts.helpers.text_analysis_helpers import get_date_today_formatted, add_import_time_columns, get_s3_location, \
+    PARTITION_KEYS, get_glue_env_var
+from scripts.helpers.housing_mmh_vulnerability_keywords import damp_mould, health, children, elderly, disabled, finance, \
+    death, home, immigration, crime, asb, domestic_violence, word_lists
+
+
+def remove_punctuation(text):
+    return text.translate(str.maketrans('', '', punctuation))
+
+
+def remove_stopwords(text):
+    return " ".join([word for word in str(text).split() if word not in STOP_WORDS])
+
+
+def find_keyword(text):
+    if any(word in word_lists for word in text.split()):
+        return 1
+    else:
+        return 0
+
+
+def main():
+    # Set up logging
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
+
+    # parameters
+    glue_client = boto3.client('glue')
+    s3_location_mmh = get_glue_env_var("s3_source_mtfh_notes")
+    mmh_case_notes = get_glue_env_var('source_table_mtfh_notes')
+    s3_location_tenure = get_glue_env_var('s3_source_tenure')
+    tenure_reshaped = get_glue_env_var('source_table_tenure')
+    s3_output_location = get_glue_env_var('s3_output_path')
```

Comment:
Better to move the setup of logging and import all environmental variables to the top, just after important dependencies (line 9). In order to avoid set up the logging again and reading these variables every time the function is invoked.